projects/packages/my-jetpack/src/class-initializer.php

```diff
@@ -65,6 +65,7 @@ class Initializer {
 	const VIDEOPRESS_STATS_KEY                           = 'my-jetpack-videopress-stats';
 	const VIDEOPRESS_PERIOD_KEY                          = 'my-jetpack-videopress-period';
 	const MY_JETPACK_RED_BUBBLE_TRANSIENT_KEY            = 'my-jetpack-red-bubble-transient';
+	const BACKUP_STATUS_TRANSIENT_KEY                    = 'my-jetpack-backup-status';
```

Comment:
This constant is unused in this file.  Was it left here in this file unintentionally?
I believe it should be removed. 😉

Reply:
Ooh nice catch, thanks I'll remove that!